src/main/resources/MapMacro/Code/Configuration.xml

```diff
@@ -75,7 +75,7 @@
       </apiKey>
     </class>
     <property>
-      <apiKey/>
+      <apiKey>AIzaSyBuiJnKVxb9XPoSF8Rv8XC6sZBMzidQPF8</apiKey>
```

Comment:
I think we shouldn't commit any key here, since it has to be linked to a google account. Is this your key?